datafusion/sqllogictest/test_files/explain_tree.slt

```diff
@@ -246,6 +251,69 @@ physical_plan
 11)│        format: csv        │
 12)└───────────────────────────┘
 
+query TT
+explain VERBOSE SELECT * FROM limit_table LIMIT 10;
```

Comment:
why does this use `EXPLAIN VERBOSE`? All the rest of the queries in this file just use `EXPLAIN` 🤔 

I think it would be better to do the same

```suggestion
explain SELECT * FROM limit_table LIMIT 10;
```